### PR TITLE
JWWEB-425 Integrate etherscan proxy

### PR DESCRIPTION
**Pull request by IViol**
**Note**: feature can't be tested locally because CORS header is not present in API server response

### CODE REVIEW

File: flow-typed/config.js [hunk 1]
```diff
@@ -14,6 +14,9 @@ declare type JSONFormatConfig = {|
   +size: number,
 |}
 
+type BlockExplorerAPILink = 'https://api-blockchain-service.jwallet.network' |
+  'http://api-blockchain-service.jwallet.develop.stacks.jdev.network'
```
Comment (flexxnn) [inline flow-typed/config.js]: It is our own internal development infrastructure, I think it is not good to leave it in opensource.
Let's discuss it offline

File: src/config.js [hunk 2]
```diff
@@ -107,10 +107,11 @@ const config: AppConfig = {
...
   walletIdInitTimeout: 100,
 
-  /**
-   * blockexplorer url
-   */
-  blockExplorerLink: 'etherscan.io',
+  blockExplorerUILink: 'etherscan.io',
+
+  blockExplorerAPILink: __DEV__
+    ? 'http://api-blockchain-service.jwallet.develop.stacks.jdev.network'
```
Comment (flexxnn) [inline src/config.js]: ^^ My comment above

File: flow-typed/config.js (hunk 1 above)
Comment (stcherenkov) [reply flow-typed/config.js]: It is already in opensource. ;)

Comment (stcherenkov) [reply flow-typed/config.js]: I think, we should move these values to environment variables so they could be set in run-time. Like those `[MAIN_RPC_ADDR]` in index.html

Comment (stcherenkov) [reply flow-typed/config.js]: But in types it should be probably just string. There is no need for enum with configuration variables, it feels a little bit off...

Comment (IViol) [reply flow-typed/config.js]: At least for localhost dev we need default value without any params from docker image

Comment (stcherenkov) [reply flow-typed/config.js]: Yep, but we can pass env variables through webpack, without using Docker

Comment (stcherenkov) [reply flow-typed/config.js]: See here: https://github.com/jibrelnetwork/jwallet-web/blob/develop/webpack.config.js#L266-L271

**Pull request merged**